qml/StatisticsChartView.qml

```diff
@@ -262,8 +269,43 @@ ChartView {
     function setXAxis(min, max, force = false) {
         // If axes has been set manually and forced is not set, do not change
         if (force || !manuallySetAxes) {
-            dateTimeAxisX.min = min
-            dateTimeAxisX.max = max
+            var axesLimits = checkAxes(min, max)
+            dateTimeAxisX.min = axesLimits.min
+            dateTimeAxisX.max = axesLimits.max
+        }
+    }
+
+    function checkAxes(min, max) {
+        var axisMin, axisMax
+        if (min == y_MIN_DEFAULT || min == y_MAX_DEFAULT) 
+        {
+            axisMin = 0
+        }
+        else
+        {
+            axisMin = min
+        }
+        if (max == y_MIN_DEFAULT || max == y_MAX_DEFAULT) 
+        {
+            axisMax = 1
```

Comment:
Y would use also 0. So if no data, -1 : 1 is shown. @juanlofer-eprosima @rsanchez15 ?